### DISCUSSION TITLE
3D printed enclosure.

### DISCUSSION THREAD

**Discussion by necroware** [necroware/gameport-adapter#49] [Ideas]
This discussion is for ideas only. If you have some questions, please use appropriate discussions.

**Comment by JocPro** [comment]
Hi there.

I want to share a little enclosure that I modeled for the adapter. It can be closed with a couple of 1/8" x 3/4" (I think they're also called # 5-40 x 3/4") round head machine screws and the holding posts from the DB-15 port.

![image](https://user-images.githubusercontent.com/16336845/223713914-97d72a5c-210f-4b46-b4fe-703ad8dc3ffc.png)
![image](https://user-images.githubusercontent.com/16336845/223714020-4cf45c84-955e-4d6b-80da-42bc43afc0d8.png)

 It fits the Type-C Arduino Pro Micro boards, but I'll try to modify it in the near future to fit the Micro-B ones, as they're a little shorter. I printed it with an Ender 3 v2 printer in PLA with 0.2mm layers with a 0.4mm nozzle and fits the board like a glove, with enough clearance for the solder joints in the back of the board.

[Gameport Adapter Case - STLs.zip](https://github.com/necroware/gameport-adapter/files/10920628/Gameport.Adapter.Case.-.STLs.zip)

Feel free to share it.

**Reply by JocPro** [reply]
Here's the Micro B variant for the top, and both side by side.

![image](https://user-images.githubusercontent.com/16336845/223967876-b27b39bc-10be-430c-a14d-81beb8645373.png)
![image](https://user-images.githubusercontent.com/16336845/223968218-79bd3970-c345-4d13-8dc4-f988a599101d.png)

[Gameport Adapter Case - Micro B top.zip](https://github.com/necroware/gameport-adapter/files/10929508/Gameport.Adapter.Case.-.Micro.B.top.zip)

**Reply by necroware** [reply]
Nice! I would make the hole around the DB15 port slightly wider, since some cheaper joysticks have a thick plastic connector without screws, which wouldn't fully pass. I think, 2mm around the port should be enough.

**Reply by JocPro** [reply]
![image](https://user-images.githubusercontent.com/16336845/224035266-68d48b08-1f44-4cea-bd94-e9c38de03858.png)

I didn't take this kind of port in consideration; so here's an improved clearance version!

![image](https://user-images.githubusercontent.com/16336845/224035412-f4059650-ed58-4780-82d8-15e01ab91270.png)

[Gameport Adapter Case (Clearance).zip](https://github.com/necroware/gameport-adapter/files/10932273/Gameport.Adapter.Case.Clearance.zip)

**Comment by FreakazoidFr** [comment]
Hi everyone,
I want to make the gameport adapter because I have 2 old MS Sidewinder pads I want to use again.
Here is my questions : does the adapter works with the first generation of sidewinder and if yes, can I use link mode / pass-through (to connect one pad on another pad) ?
Thanks for your amazing work !

**Reply by necroware** [reply]
Hi, I created a separate discussion for general Q&A, so we all can find everything after a while. Please use that for the questions in the future, or create new discussions with questions if you think, that it is worth it. Regarding your question you can find a list of supported devices in the documentation. There are also videos about this project, where the Sidewinder Pad is mentioned explicitly. Pass-through is not supported, because the effort is quite big, but the benefit is very low, since every PC has a lot of USB ports and you can just use one adapter per input device. Pass-through was a thing back then, because PCs usually had only one GamePort, with USB you don't have that limitation.

**Comment by mjack3k** [comment]
I made one for the USB-C board - I think it was a bit longer than the microUSB version. 

Snap-fit, no screws required, around 1h print for whole thing. 

Recommend 0.2mm layer

[Link](https://www.printables.com/model/455098-case-for-gameport-adapter-by-necroware)
![20230417_204834.jpg](https://user-images.githubusercontent.com/128301824/232586583-75956f62-3a89-4a86-8868-aeef9a764263.jpg)

**Reply by mrc333777** [reply]
can you make me one of these? the whole thing with the case?

**Comment by MacGH23** [comment]
You can use this one, too.
http://www.yard2usb.de
I got one myself
